e2etest/bgp_test.go

```diff
@@ -229,15 +225,19 @@ var _ = ginkgo.Describe("BGP", func() {
 
 			var peers []peer
 			var peerAddrs []string
-			for _, c := range frrContainers {
+			for i, c := range frrContainers {
```

Comment:
General comment about the commit: `Filling ... in the external containers but using them as
configuration of speakers is misleading.` - that's misleading because it wasn't used here correctly :sweat_smile:
From the `containerConfig`'s (which represents the external FRR container) perspective, configuration for the speaker side should be under `NeighborConfig` (and not `RouterConfig`). That's because `RouterConfig` is how it should configure itself and `NeighborConfig` is the way the speaker should present itself to the container.

I think we should preserve that and not declare any parameters somewhere else.

Reply:
I think if we call the structure FRR container, a reader expects to find parameters affecting the container only.
We are currently leveraging some of those to set up the metallb config because they are symmetric (i.e. the password, the ASNs). Some other belong to metallb only and it does not make sense to have them here unless we think about a wider wrapper.